resources/less/main.less

```diff
@@ -1,7 +1,6 @@
 // Bootstrap UI demo
 
-@import "bootstrap.ui/src/less/bootstrap-ui";
+@import "../../node_modules/bootstrap-ui/src/less/bootstrap-ui";
```

Comment:
This can stay without the `../../node_modules/` part. Grunt can find the destination thanks to `path` option in `grunt/less.js`.